tests/src/test/scala/org/apache/openwhisk/standalone/StandaloneServerFixture.scala

```diff
@@ -93,6 +95,7 @@ trait StandaloneServerFixture extends TestSuite with BeforeAndAfterAll with Stre
       manifestFile.foreach(FileUtils.deleteQuietly)
       serverProcess.destroy()
     }
+    FileUtils.forceDelete(new File(dataDirPath))
```

Comment:
I guess this answers my question from above regarding data deletion when the process exits...

Reply:
At least for tests...